### ISSUE TITLE
api for a kappa-view

### ISSUE THREAD

**Issue by dominictarr** [kappa-db/kappa-view#1]
I'm trying to understand how I might create an adapter that allows flumeviews and kappaviews to be interchangeable. 

in flume, views expose a `createSink` method, and a `since` observable.

in kappa, it looks like the batch method is used, which takes a callback, but doesn't expose where it's up to exactly. Or does it in a way I havn't figured out yet?

I also noticed that `multifeed-index` accepts ops that have both keys and sequences - it indexes many logs not just one log. It seems like this might ruin my plan to make an adapter, because flume assumes a single log (which in ssb is many logs combined, in receive order). I guess it would be easy to adapt a kappa view to flume, because if the view can support many logs, then it can support just one.

**Comment by hackergrrl** [comment]
You're right: it doesn't expose indexing progress right now, at least not directly. However, you could [deserialize](https://github.com/kappa-db/multifeed-index/blob/master/lib/state.js#L35) the opaque state object given to the `storeState` userland callback to figure it out if you really needed to.

Yeah, this is a tricky problem, since flumedb hands off the multiple feeds problem to ssb, but they're first-class in kappa..
